src/biz/metrics.go

```diff
@@ -38,10 +38,10 @@ type Network raw.Network
 
 // BlkIO stands for Block I/O stats
 type BlkIO struct {
-	TotalReadCount  float64
-	TotalWriteCount float64
-	TotalReadBytes  float64
-	TotalWriteBytes float64
+	TotalReadCount  *float64
+	TotalWriteCount *float64
+	TotalReadBytes  *float64
+	TotalWriteBytes *float64
```

Comment:
Since we don't control how values are encoded we cannot use json tags annotations instead of nil values, right?

Example:
```
TotalReadCount float64 `json:",omitempty"`
```

Reply:
Yep, they're different structs...

Reply:
As discussed _offline_, it is not the same struct, so it is not possible 🤐